components/forms/Form.js

```diff
@@ -1,43 +1,47 @@
 import styled from "styled-components";
 
-export default function WeightForm({ onAddWeight }) {
+export default function Form({ onAddData }) {
   function sendForm(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
-    const { weight, date } = Object.fromEntries(formData);
+    const { weight, date, height } = Object.fromEntries(formData);
 
-    onAddWeight(weight, date);
+    onAddData(weight, date, height);
```

Comment:
It could be beneficial for you to reset/clear the form after every submit event.

Reply:
thanks for the idea, just implemented that :)